tests/test_estimation.py

```diff
@@ -1,91 +1,206 @@
 import numpy as np
-from pyriemann.estimation import (Covariances, ERPCovariances,
-                                  XdawnCovariances, CospCovariances,
-                                  HankelCovariances, Coherences, Shrinkage)
+from pyriemann.estimation import (
+    Covariances,
+    ERPCovariances,
+    XdawnCovariances,
+    CospCovariances,
+    HankelCovariances,
+    Coherences,
+    Shrinkage,
+)
 import pytest
 
 
-def test_covariances():
+estim = ["cov", "scm", "lwf", "oas", "mcd", "corr"]
```

Comment:
What about `sch` estimator?

Reply:
yes, I need to rebase.

Reply:
and to merge schafer estimator 🤦‍♂️

Reply:
Yes! But try to rebase this branch, rather than merge master into the branch...